packages/node/src/policy-gen/policy-gen-context.js

```diff
@@ -95,19 +88,35 @@ export class PolicyGeneratorContext {
    * A mapping of compartment names to missing module specifiers encountered
    * during the policy generation process.
    *
-   * @type {MissingModuleMap | undefined}
+   * @type {Readonly<Set<string>>}
    */
 
   #missingModules
 
-  /** @type {Loggerr} */
+  /** @type {Readonly<Loggerr>} */
   #log
 
   /**
    * Used for reading source files
    */
   static #decoder = new TextDecoder()
 
+  /**
+   * Cache of specifier to filepath (or lack thereof)
+   *
+   * `null` is used to differentiate the value if the specifier has no filepath
+   * vs. if it is actually absent from the `Map` (as in the case of a builtin).
```

Comment:
Hard to understand which of the situation is indicated by null and which is undefined. Made my guess and wrote:

```suggestion
   * `null` when the specifier has no filepath
   * `undefined` if it is actually absent from the `Map` (as in the case of a builtin).
```

Reply:
- [ ] rewrite comment